kythe/proto/storage.proto

```diff
@@ -91,7 +91,7 @@ message VNameMask {
   bool language = 5;
 }
 
-// An Entry associates a fact with a graph object (node or edge).  This is the
+// An Entry associates a fact with a graph object (node XOR edge).  This is the
```

Comment:
This is a reasonable choice, but we intentionally left room in the data model for edges to carry facts. We chose not to use that (and collapsed the ordinal case into the edge label to avoid the one place where we used to rely on it). But it's not intrinsic in the model.

Reply:
We've been pretty consistent on not wanting facts on edges.  We're probably at the point we can just declare it as truth.

Reply:
Sure, but I think if we want to be explicit about that, we should document more than simply "or" becoming "xor". Probably the storage model doc could use some colour too.

Reply:
https://github.com/google/kythe/pull/3109/commits/c783b50d9e8a8ca7f0b238e1c8dda5b6251823f0

?